src/OptionManager.php

```diff
@@ -312,7 +313,13 @@
 <br>
 <table cellpadding="3" width="30%" align="center">
 
+
 <?php
+    $ids = SystemConfig::getValue('sInactiveClasification');
```

Comment:
I believe this needs to be updated now

```suggestion
    $ids = SystemConfig::getValue('sInactiveClassification');
```